model/BookletClassService.php

```diff
@@ -28,7 +28,8 @@
 class BookletClassService extends tao_models_classes_ClassService
 {
     const CLASS_URI = 'http://www.tao.lu/Ontologies/Booklet.rdf#Booklet';
-    
+    const GROUP_PROPERTY_URI = 'http://www.tao.lu/Ontologies/TAOGroup.rdf#Booklets';
```

Comment:
Check with @llecaque if the URIs are used as class constants or needs to be defined as plain old constants in `includes/constants.php`